AppCreationScripts/sample.json

```diff
@@ -14,7 +14,7 @@
   "AADApps": [
     {
       "Id": "client",
-      "Name": "up-console",
+      "Name": "active-directory-dotnetcore-console-up-v2",
```

Comment:
The goal of this Name is not to be the name of the repository (there is a "RepositoryUrl" property for that), but the name that will appear in the Readme.md.
I would prefer we keep up-console.
What is your concern @kalyankrishna1 ?